2008-08-02  Robert Millan  <rmh@aybabtu.com>
 
	* util/i386/pc/grub-install.in: Remove trailing slash from prefix.
 
--- util/i386/pc/grub-install.in~	2008-07-13 20:13:36.000000000 +0200
+++ util/i386/pc/grub-install.in	2008-10-29 20:16:31.000000000 +0100
@@ -241,7 +241,7 @@
 modules="$modules $fs_module $partmap_module biosdisk $devabstraction_module"
 
 $grub_mkimage --output=${grubdir}/core.img \
-    --prefix=${prefix_drive}`make_system_path_relative_to_its_root ${grubdir}`/ \
+    --prefix=${prefix_drive}`make_system_path_relative_to_its_root ${grubdir}` \
     $modules || exit 1
 
 # Now perform the installation.
